app/test_main.py

```diff
@@ -1,3 +1,11 @@
 from app.main import get_human_age
 
-# write your code here
+def test_should_return_correct_result() -> None:
+    assert (get_human_age(0, 0) == [0, 0]
+            and get_human_age(14, 14) == [0, 0]
+            and get_human_age(15, 15) == [1, 1]
+            and get_human_age(23, 23) == [1, 1]
+            and get_human_age(24, 24) == [2, 2]
+            and get_human_age(27, 27) == [2, 2]
+            and get_human_age(28, 28) == [3, 2]
```

Comment:
The expected result for `get_human_age(28, 28)` should be `[3, 3]` instead of `[3, 2]`. This is because the conversion logic for dogs should also add another year after 27, similar to cats.